src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingDeclarationTests.cs

```diff
@@ -1626,39 +1623,6 @@ static async Task Main()
             CompileAndVerify(compilation, expectedOutput: "Dispose async");
         }
 
-        [Fact]
-        public void UsingDeclarationAsyncPattern()
```

Comment:
It'd be great if you commented on the tests you removed, or maybe your general strategy (when did you update vs. remove?).

Reply:
Generally I removed the tests that were not possible to change. For instance anything async pattern is no longer valid: you can't have a ref struct in an async method, so we can never implement async dispose via a pattern.

Things that were possible if it was a ref struct were changed to have the disposer be a ref struct, and other tests that used patterns where they weren't strictly required were updated to implement via interface